src/Components/HistoricalDataChart/HistoricalDataChart.scss

```
@@ -1,36 +1,17 @@
-.chartContainer{
-   display: flex;
-   flex-direction: column;
-   align-items: flex-start;
-   position: absolute;
-   top: 0;
-   right: 0;
-   margin-right: 8%;
-}
-
-.dropdownContainer {
-   margin-right: 200px;
-   align-self: flex-end;
-}
-
-.dateDropdown{
-   padding-left: 8px;
-}
+.chartDateFilter {
```

@kgaikwad since the date dropdown and historical chart are part of same container, it is making dropdown & info icon non-clickable. One of my suggestion would be as below.  I saw the mockup shared by Stefan and I think it will be good to add bit of right margin to the chart to have some space between chart and dropdown. Let me know your thoughts! 


```suggestion
.chartDateFilter {
   position: relative;
   z-index: 10;
   margin-left: 48px;
```

---

Good catch! :+1:  Fixed!